code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -552,23 +553,6 @@
 	return ..()
 
 
-/mob/living/simple_animal/update_mobility(value_otherwise = TRUE)
-	if(HAS_TRAIT_NOT_FROM(src, TRAIT_IMMOBILIZED, BUCKLED_TRAIT))
-		drop_all_held_items()
-		mobility_flags = NONE
-	else if(buckled)
-		mobility_flags = MOBILITY_FLAGS_INTERACTION
-	else
-		if(value_otherwise)
-			mobility_flags = MOBILITY_FLAGS_DEFAULT
-		else
-			mobility_flags = NONE
-	if(!(mobility_flags & MOBILITY_MOVE))
-		walk(src, 0) //stop mid walk
-
-	update_transform()
-	update_action_buttons_icon()
```

Comment:
This functionality moved over I assume, nother note

Reply:
Yes, this is the sort of change I expect a TM to find errors from.
Instead of having six separate custom mobility logics for each kind of living mob I'm trying to get us a standard logic for all of them that can be redefined in more specific ways, but makes use of the same base system.